pkg/morph/client/client.go

```diff
@@ -367,23 +367,50 @@ func (c *Client) TestInvoke(contract util.Uint160, method string, args ...any) (
 
 // TestInvokeIterator is the same [Client.TestInvoke] but expands an iterator placed
 // on the stack. Returned values are the values an iterator provides.
-func (c *Client) TestInvokeIterator(contract util.Uint160, method string, args ...any) (res []stackitem.Item, err error) {
+// If prefetchElements > 0, that many elements are tried to be placed on stack without
+// additional network communication (without the iterator expansion).
+func (c *Client) TestInvokeIterator(contract util.Uint160, method string, prefetchElements int, args ...any) (res []stackitem.Item, err error) {
 	c.switchLock.RLock()
 	defer c.switchLock.RUnlock()
 
 	if c.inactive {
 		return nil, ErrConnectionLost
 	}
 
-	sid, iter, err := unwrap.SessionIterator(c.rpcActor.Call(contract, method, args...))
-	if err != nil {
-		return nil, fmt.Errorf("iterator expansion: %w", err)
+	var sid uuid.UUID
+	var iter result.Iterator
+	items := make([]stackitem.Item, 0)
```

Comment:
In the prefetching case this is a useless allocation.

Reply:
fixed